code/game/objects/items/weapons/dnascrambler.dm

```diff
@@ -22,16 +22,20 @@
 		if(src.used)
 			return
 
+		if(M.get_species() == "Machine")
```

Comment:
Is this because its cheaper to look up vs a string? Or whats the reasoning behind this?